engine/function/src/templates/Numeric.ftl

```diff
@@ -1488,6 +1488,7 @@ public class Numeric {
         }
 
         double sum = 0;
+        long nullCount = 0;
```

Comment:
Do we want to handle infinities the way `aggBy` does?
```
    private double currentValueWithSum(long totalNormalCount, long totalNanCount, long totalPositiveInfinityCount,
            long totalNegativeInfinityCount, double newSum) {
        if (totalNanCount > 0 || (totalPositiveInfinityCount > 0 && totalNegativeInfinityCount > 0)) {
            return Double.NaN;
        }
        if (totalNegativeInfinityCount > 0) {
            return Double.NEGATIVE_INFINITY;
        }
        if (totalPositiveInfinityCount > 0) {
            return Double.POSITIVE_INFINITY;
        }
        if (totalNormalCount == 0) {
            return QueryConstants.NULL_DOUBLE;
        }
        return (double) newSum;
    }
```

Reply:
Larry and I had a discussion on this before the prior fix.  It is unnecessary because of how IEEE754 does things.  Look at the unit tests.